agogosml/agogosml/tools/sender.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""Script to push inputs to an agogosml client."""
+from argparse import ArgumentParser
+from argparse import ArgumentTypeError
+from argparse import FileType
```

Comment:
Nit: Any reason why we shouldn't collapse these imports into one statement?

Reply:
Makes for cleaner diffs. Strongly recommended IMO. I'll add the isort linter later to enforce consistent import styles.

Reply:
See PR https://github.com/Microsoft/agogosml/pull/217.